apps/src/code-studio/components/aiTutor/aiTutorPanel.tsx

```diff
@@ -14,6 +14,11 @@ interface AITutorPanelProps {
 const AITutorPanel: React.FunctionComponent<AITutorPanelProps> = ({open}) => {
   const level = useAppSelector(state => state.aiTutor.level);
   const isAssessmentLevel = level?.isAssessment;
+  const isAvailable = level?.aiTutorAvailable;
+  let aiTutorAvailable = true;
```

Comment:
I think we can just use
```
const aiTutorAvailable = !isAssessmentLevel && isAvailable;
```
Does that seem right?

Reply:
I wonder if we can avoid confusion with variable names by using the ones below...
```
const isAssessmentLevel = level?.isAssessment;
const aiTutorAvailable = level?.aiTutorAvailable;
const renderAITutor = !isAssessmentLevel && isAvailable;
```

Reply:
Did it!